### DISCUSSION TITLE
What's the difference between IMG_1234.JPG vs IMG_1234-98765.JPG?

### DISCUSSION THREAD

**Discussion by BrendonGames** [icloud-photos-downloader/icloud_photos_downloader#872] [Q&A] [answered]
After download I use Immich to manage my photos, but I've noticed that there are a lot of duplicated photos that only differ in filename. 
One has the original name (IMG_1234.JPG or filename.JPG) and the other has 5 numbers added at the end (IMG_1234-98765 or filename-98765.JPG). What's the difference between them and if they're the same, how do I stop downloading them?

![image](https://github.com/icloud-photos-downloader/icloud_photos_downloader/assets/89480896/ce0d7b34-182b-4ca1-aa81-547f7ec2904d)

**Comment by AndreyNikiforov** [comment, marked as answer]
I added [a section to the doc explaining the de-duplication strategies](https://icloud-photos-downloader.github.io/icloud_photos_downloader/naming.html#duplicates). Pls see if that answers your question.

**Reply by BrendonGames** [reply]
That seems to answer it, I also added --until-found to prevent it from trying to download again. Thank you
